gqlgen.yml

```diff
@@ -25,6 +25,9 @@ models:
     model: github.com/evergreen-ci/evergreen/rest/model.FileDiff
   Task:
     model: github.com/evergreen-ci/evergreen/rest/model.APITask
+    fields:
+      dependsOn:
```

Comment:
I didn't realize this is the same name as a field in the task API model, but has a different structure. Could you name this something else?